bluepyopt/ephys/morphologies.py

```diff
@@ -94,7 +95,12 @@ def instantiate(self, sim=None, icell=None):
         # TODO this is to get rid of stdout print of neuron
         # probably should be more intelligent here, and filter out the
         # lines we don't want
-        sim.neuron.h.hoc_stdout('NUL')
+
+        if platform.system()=='Windows':
```

Comment:
@wvangeit sorry about skimping on platform-dependence.
hopefully this clears it up. my bad.

also, may need to write more unit tests for this class too as travis didn't fail for the previous commit :/